### PR TITLE
argtable2: add argtable2/2.13 recipe

### PR DESCRIPTION
**Pull request by madebr**
Specify library name and version:  **argtable2/2.13**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/argtable2/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            "--enable-shared={}".format(yes_no(self.options.shared)),
+            "--enable-static={}".format(yes_no(not self.options.shared)),
+        ]
+        self._autotools.configure(args=conf_args, configure_dir=self._source_subfolder)
+        return self._autotools
+
+    def _run_nmake(self, target):
+        autotools = AutoToolsBuildEnvironment(self)
+        autotools.libs = []
+        vars = " ".join("CONAN_{}=\"{}\"".format(k, v) for k, v in autotools.vars.items())
```
Comment (prince-chrismc) [inline recipes/argtable2/all/conanfile.py:67]: what is `k` and `v`?

**Review by prince-chrismc — APPROVED** [review]
LGTM but this autotools is still new to me

File: recipes/argtable2/all/conanfile.py (hunk 1 above)
Comment (madebr) [reply recipes/argtable2/all/conanfile.py:67]: short for key and value

**Comment by madebr** [comment]
> LGTM but this autotools is still new to me

For msvc, I use the `CFLAGS/CPPFLAGS/CXXFLAGS/LDFLAGS` that are calculated by `AutotoolsBuildEnvironment`.

**Comment by prince-chrismc** [comment]
> For msvc, I use the `CFLAGS/CPPFLAGS/CXXFLAGS/LDFLAGS` that are calculated by `AutotoolsBuildEnvironment`.

Ahhh! I see it now, you are replacing the standard variables with a conan prefix to re use them in the original. I presume to avoid collisions with those already being used by upstream.

The Conan build wrappers do a lot of heavy lifting, which is amazing, but I dont understand (from having never spent hours debugging) how they work.

Thank you for explaining :+1:

**Review by uilianries — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Pull request merged**